src/api.ts

```diff
@@ -2,7 +2,9 @@ import * as vscode from 'vscode';
 import { performAction } from './actions/client';
 import { ActionsController } from './actions/controller';
 import { ExecuteAction, SnippetAction, CommandAction, FileAction } from './actions/impl';
+import { getLogger } from './logger/logger';
 export * from "./actions/interfaces";
+import _ = require('lodash');
```

Comment:
I recommend you only import the exact functions you need from lodash
You can use an ESM compatible lodash build from here: https://www.npmjs.com/package/lodash-es

Reply:
import {isEmpty, get} from 'lodash';
?
(We have _ in all our projects..)

Reply:
yes, but that was before ESM imports
`import {foo, bar, bamba} from "../blah"`